Gemfile

```diff
@@ -3,73 +3,42 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.6.0'
 
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'bootstrap', '~> 4.3.1'
+gem 'rubocop', '~> 0.68.1', require: false
```

Comment:
Also, add all the gems I mention in the practice

Reply:
For this particular gem (rubocop), create 2 files `.rubocop-pernix.yml` and `.rubocop.yml`, you have to add them to the project's main folder (where the Gemfile, README and all those files are). You can copy the content for those files from the CACISA project.